Branch-SDK/Branch-SDK/Requests/BranchInstallRequest.m

```diff
@@ -48,7 +49,14 @@ - (void)makeRequest:(BNCServerInterface *)serverInterface key:(NSString *)key ca
     params[BRANCH_REQUEST_KEY_IS_REFERRABLE] = @(preferenceHelper.isReferrable);
     params[BRANCH_REQUEST_KEY_DEBUG] = @(preferenceHelper.isDebug);
     
-    [serverInterface postRequest:params url:[preferenceHelper getAPIURL:BRANCH_REQUEST_ENDPOINT_INSTALL] key:key callback:callback];
+    if ([[BNCStrongMatchHelper strongMatchHelper] shouldDelayInstallRequest]) {
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(500 * NSEC_PER_MSEC)), dispatch_get_main_queue(), ^{
+            [serverInterface postRequest:params url:[preferenceHelper getAPIURL:BRANCH_REQUEST_ENDPOINT_INSTALL] key:key callback:callback];
```

Comment:
> it will almost always finish first

Read: be prepared for race conditions.

Naturally, your choice to do so, but this may lead to issues. Fortunately, this request is fairly insignificant -- if it fails, life goes on, and the only loss is potentially 100% match (which falls to probably what, 95+?).

If this request was required, this would not work, but for the role is fills, I think it'll be ok.